libs/vertexai/langchain_google_vertexai/functions_utils.py

```diff
@@ -54,7 +54,7 @@ def _format_tool_to_vertex_function(tool: BaseTool) -> FunctionDescription:
                     }
                     for k, v in schema["properties"].items()
                 },
-                "required": schema["required"],
+                "required": schema.get("required", []),
```

Comment:
should we update line #34 too?

Reply:
Wraped both blocks into a function, as it was duplicated code and make it so that if there is no required there's no key